src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String name;
+
+    public Fruit(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
```

Comment:
you should override equals() and hashCode() in order to store in as a key in your map